docs/ops/external_resources.zh.md

```diff
@@ -234,127 +230,122 @@ class FPGAInfo extends ExternalResourceInfo {
 </div>
 </div>
 
-Create a file with name `org.apache.flink.api.common.externalresource.ExternalResourceDriverFactory` in `META-INF/services/`
-and write the factory class name (e.g. `your.domain.FPGADriverFactory`) to it.
+在 `META-INF/services/` 中创建名为 `org.apache.flink.api.common.externalresource.ExternalResourceDriverFactory` 的文件，向其中
+写入工厂类名，如 `your.domain.FPGADriverFactory`。
 
-Then, create a jar which includes `FPGADriver`, `FPGADriverFactory`, `META-INF/services/` and all the external dependencies.
-Make a directory in `plugins/` of your Flink distribution with an arbitrary name, e.g. "fpga", and put the jar into this directory.
```

Comment:
Does the created directory need to have the same name as resource name configured in `external-resources`?

Reply:
No, the name of the directory has no influence on the Plugin Mechanism.